astropy/io/misc/tests/test_hdf5.py

```diff
@@ -707,7 +707,7 @@ def assert_objects_equal(obj1, obj2, attrs, compare_class=True):
 
 
 @pytest.mark.skipif(not HAS_H5PY, reason="requires h5py")
-def test_hdf5_mixins_qtable_to_table(tmp_path):
+def code(tmp_path):
```

Comment:
Why this change?

Reply:
wow, that's a syntaxically valid mistake I must have made by trying to open vscode from another editor and typing in the wrong tab. Thanks for spotting it